### DISCUSSION TITLE
Documentation: Missing a section explaining how to update Flux.

### DISCUSSION THREAD

**Discussion by lcallif** [livewire/flux#468] [Feature Requests] [answered]
The documentation includes instructions on initial installation, but is missing a section showing how to update Flux.

**Comment by joshhanley** [comment]
@lcallif don't take this the wrong way, but why is this needed? I'm just curious, as Livewire and Laravel docs don't have instructions on how to use composer to update them.

**Reply by lcallif** [reply]
Nope, not taking this the wrong way.

Maybe the others should have it...  It makes it easier for the people that need to update their app, such as Flux, know the command is to perform the update.  

Also, just because others don't, doesn't mean Flux shouldn't.  Does Flux do something that others don't ?  Maybe it shouldn't document it...  :-)

Nope, not needed.  Nice to have, yes.

**Comment by pitbully** [comment, marked as answer]
This is already done in the documentation.
https://fluxui.dev/docs/installation

Scroll down to “Keeping Flux updated”

**Reply by lcallif** [reply]
Ohhh myyyy.  Thank you!

**Reply by joshhanley** [reply]
Ah so it is! Thanks @pitbully 😁
